server/api/api/endpoints/pipelines.py

```diff
@@ -42,6 +42,7 @@ async def list_pipelines(
     sort_by: str = "",
     page_token: str = "",
     filter_: str = Query("", alias="filter"),
+    name__contains: str = Query("", alias="name--contains"),
```

Comment:
Why the double dash? Could use `name-contains` for this param name.

Reply:
Some ORMs like to split variable names from operation names with a clear separator (one that hardly ever would be part of a variable name). But I do see that we don't do this on MLRun, so it'd just be a noisy addition to the codebase. 

I removed it for us.